src/config/sites_fixes_v2.json

```diff
@@ -247,6 +247,14 @@
             "rules": [
                 "#watch-appbar-playlist * { -webkit-filter: none !important; }"
             ]
+        },
+        {
+            "url": [
```

Comment:
Dark Reader support wildcards like it is made for [Google](https://github.com/BANOnotIT/darkreader/blob/2d5d38bf172ea405c4437f92a96f10325cde5db5/src/config/sites_fixes_v2.json#L52). There's also "music.yandex.kz" and there may be more domains.

Please, place "music.yandex" config after "mail.live.com" so it will be easier to maintain the growing site list.